### ISSUE TITLE
Nautobot 2.0 Migration error: `AttributeError: 'NoneType' object has no attribute 'name'`

### ISSUE THREAD

**Issue by joewesch** [nautobot/nautobot-plugin-bgp-models#156]
### Environment
* Python version:  3.9
* Nautobot version:  2.0.3
* nautobot-bgp-models version: 0.20.0

### Expected Behavior
`nautobot-server post_upgrade` to complete successfully

### Observed Behavior
```
Applying nautobot_bgp_models.0005_use_upstream_role_part2...Traceback (most recent call last):
  File "/opt/nautobot/bin/nautobot-server", line 8, in <module>
    sys.exit(main())
  File "/opt/nautobot/lib64/python3.9/site-packages/nautobot/core/cli/__init__.py", line 54, in main
    run_app(
  File "/opt/nautobot/lib64/python3.9/site-packages/nautobot/core/runner/runner.py", line 297, in run_app
    management.execute_from_command_line([runner_name, command] + command_args)
  File "/opt/nautobot/lib64/python3.9/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
    utility.execute()
  File "/opt/nautobot/lib64/python3.9/site-packages/django/core/management/__init__.py", line 413, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/opt/nautobot/lib64/python3.9/site-packages/django/core/management/base.py", line 354, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/opt/nautobot/lib64/python3.9/site-packages/django/core/management/base.py", line 398, in execute
    output = self.handle(*args, **options)
  File "/opt/nautobot/lib64/python3.9/site-packages/nautobot/core/management/commands/post_upgrade.py", line 100, in handle
    call_command(
  File "/opt/nautobot/lib64/python3.9/site-packages/django/core/management/__init__.py", line 181, in call_command
    return command.execute(*args, **defaults)
  File "/opt/nautobot/lib64/python3.9/site-packages/django/core/management/base.py", line 398, in execute
    output = self.handle(*args, **options)
  File "/opt/nautobot/lib64/python3.9/site-packages/django/core/management/base.py", line 89, in wrapped
    res = handle_func(*args, **kwargs)
  File "/opt/nautobot/lib64/python3.9/site-packages/django/core/management/commands/migrate.py", line 244, in handle
    post_migrate_state = executor.migrate(
  File "/opt/nautobot/lib64/python3.9/site-packages/django/db/migrations/executor.py", line 117, in migrate
    state = self._migrate_all_forwards(state, plan, full_plan, fake=fake, fake_initial=fake_initial)
  File "/opt/nautobot/lib64/python3.9/site-packages/django/db/migrations/executor.py", line 147, in _migrate_all_forwards
    state = self.apply_migration(state, migration, fake=fake, fake_initial=fake_initial)
  File "/opt/nautobot/lib64/python3.9/site-packages/django/db/migrations/executor.py", line 227, in apply_migration
    state = migration.apply(state, schema_editor)
  File "/opt/nautobot/lib64/python3.9/site-packages/django/db/migrations/migration.py", line 126, in apply
    operation.database_forwards(self.app_label, schema_editor, old_state, project_state)
  File "/opt/nautobot/lib64/python3.9/site-packages/django/db/migrations/operations/special.py", line 190, in database_forwards
    self.code(from_state.apps, schema_editor)
  File "/opt/nautobot/lib64/python3.9/site-packages/nautobot_bgp_models/migrations/0005_use_upstream_role_part2.py", line 33, in create_roles
    peer.role_new = NautobotRole.objects.get(name=peer.role.name)
  AttributeError: 'NoneType' object has no attribute 'name'
```

### Steps to Reproduce
1. pip install upgrade Nautobot 1.6.2 -> 2.0.3 and BGP Models from 0.9.0 -> 0.20.0
2. run `nautobot-server post_upgrade`

**Comment by glennmatthews** [comment]
https://github.com/nautobot/nautobot-plugin-bgp-models/blob/next-2.0/nautobot_bgp_models/migrations/0005_use_upstream_role_part2.py#L32

pretty sure that should be:

```python
for peer in PeerEndpoint.objects.exclude(role__isnull=True).exclude(role__name__exact=""):
```

and similar on the other two cases above

**Comment by glennmatthews** [comment]
Fixed for 0.20.1 by #157
